www/about.html

```diff
@@ -24,7 +24,10 @@
     </header>
 
     <main>
-      <a href="/">← Return to the tracker</a>
+      <a href="/">
+        <span aria-hidden="true">←</span>
```

Comment:
I think there are CSS instructios you can add to make it invisible to screenreaders though, similar tothe `aria-` attributes. But this has been a long while since I worked on these things